docker/test/stateless/run.sh

```diff
@@ -176,6 +176,25 @@ done
 setup_logs_replication
 attach_gdb_to_clickhouse || true  # FIXME: to not break old builds, clean on 2023-09-01
 
+# create minio log webhooks for both audit and server logs
+clickhouse-client --query "CREATE TABLE minio_audit_logs
+(
+    log String,
+    event_time DateTime64(9) MATERIALIZED parseDateTime64BestEffortOrZero(substring(JSONExtractRaw(log, 'time'), 2, 29), 9, 'UTC')
```

Comment:
Maybe it makes sense to add current timestamp as a fallback, wdyt?

Reply:
I think it would actually lead to more confusion as the ordering could be slightly messed up.
If there is a message without time it should be something strange and we are probably not interested in that log.

Reply:
One more thing, I saw logs from minio, that are in a format hh:mm:ss.sss, but in fact the milliseconds were always '000'. If that's the case here, I'd add another column where we'll store insertion time, at least we'll have an approximation of better ordering of events

Reply:
Fair point, but the audit logs have nanosecond precision which is not set to 0 so it should be fine.

Reply:
What about the server logs next?

Reply:
Good question and I have no idea. :smile: 
I tried to get them locally but I couldn't.
We can change the code when we get an example of the log.

Reply:
ok, sgtm, sorry for nitpicking 🙂

Reply:
No need to be sorry for doing a thorough review :smile: